podcasts/PodcastViewController.swift

```diff
@@ -998,12 +1010,16 @@ class PodcastViewController: FakeNavViewController, PodcastActionsDelegate, Sync
 
         //TODO: Add analytics based on source
 
-        Task { @MainActor [weak self] in
+        podcastFeedReloadTask = Task { @MainActor [weak self] in
```

Comment:
@leandroalonso what do you think about calling `loadPodcastInfo` if the view model says that the podcast needs to be reloaded to download new episodes?